content/docs/conditional-rendering.md

```diff
@@ -118,11 +120,11 @@ ReactDOM.render(
 
 [**Try it on CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
 
-While declaring a variable and using an `if` statement is a fine way to conditionally render a component, sometimes you might want to use a shorter syntax. There are a few ways to inline conditions in JSX, explained below.
+वेरिएबल डिक्लेअर करके और  `if`  स्टेटमेंट का यूज़ करना जिससे हम कौम्पोनॅन्ट को कंडशनल रेंडर कर सके यह अच्छी बात हे, कभी आपको छोटा शार्ट सिंटेक्स उसे करना पड़ सकता हे .यहाँ नीचे, JSX में इन लाइन कंडीशंस के कुछ पर्याय दिए हे। 
```

Comment:
```
वेरिएबल डिक्लेअर करके और  `if`  स्टेटमेंट का यूज़ करना जिससे हम कौम्पोनॅन्ट को कंडशनल रेंडर कर सके यह अच्छी बात हे
```
=>
```
वेरिएबल डिक्लेअर करते वक्त, `if` स्टेटमेंट उसे कर्कटे कौम्पोनॅन्ट को कण्डीशनली रेंडर करना सही तरीका है
```

Reply:
done

Reply:
I think i did a typo here
`वेरिएबल डिक्लेअर करते वक्त, `if` स्टेटमेंट उसे कर्कटे कौम्पोनॅन्ट को कण्डीशनली रेंडर करना सही तरीका है`
=>
`वेरिएबल डिक्लेअर करते वक्त, `if` स्टेटमेंट से कौम्पोनॅन्ट को कण्डीशनली रेंडर करना सही तरीका है`

Can you make this change.
Sorry about the typo

Reply:
done